rhobs/alerting/data_plane/prometheus.high_memory_usage_alert.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pod-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: pod_alert_high_memory_usage
+    interval: 5m
+    rules:
+    - alert: container_cpu_usage_seconds_total
+      expr: sum by (pod, namespace)(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])){}
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: pod {{ $labels.pod}} pod identified with memory usage above 90%
+        description: >-
+          Pod {{ $labels.pod }} for namespace {{ $labels.namespace }} on cluster
+          {{ $labels.source_cluster }}  cluster_name
+        alert_routing_key: '{{ $labels.namespace }}'
+        runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/sre/alert-rule-HighMemoryAlertUsage.md
+    - alert: container_memory_usage_bytes
+      expr: sum by (namespace)(container_memory_usage_bytes{container!=""})
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: Pod {{ $labels.pod }}
```

Comment:
Should this also include some mention of the memory usage test? It's just a pod reference now I think?

Reply:
no threshold in `expr` again...